### PR TITLE
add feture

### PR DESCRIPTION
**Pull request by whiteWolfx99**
# Budget app
- add linters
- add controllers
- add models
- add view
- add devise
- add splash screen
- add category screen
- add transaction screen
- add test
- add live demo [live](https://budget-app.herokuapp.com/)
- add Video presentation [Video-presentation](https://drive.google.com/file/d/1gFFXjlGxdErqRo5lpQmKLBkFSAAA0STA/view?usp=sharing)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -151,7 +151,6 @@ You can deploy this project using:
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (tresorsawasawa) [inline README.md:152]: - [ ] As required for the project, kindly make sure your project is deployed properly, the home and detail page works as expected. and A link to the deployed project is present in the README. Riight now, the link works to open the home page(i.e. authentication pages)

Comment (tresorsawasawa) [inline README.md:152]: - [ ] Kindly make sure all your pages sticks as much as possible to the the one from the **[template](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=)** design.

Comment (tresorsawasawa) [inline README.md:152]: - [ ] As required for the project, kindly make sure your project matches all the required features from the project requirements. Right now, User is only able for running for the Home page. User can Sign_up but impossible for confirmation.As result, it blocks to check other features of the project. Make sure you follow the **[design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=)** for each page and ach features works as expected as required for the **[project](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md)**.

Comment (tresorsawasawa) [inline README.md:152]: - [ ] As required for the project, kindly make sure your App has unit tests for all the most important components of the RoR application. It should also have integration tests for all the most important components of the RoR application. Right now, you only add tests to your `models`. Make sure your `Views` have integrations tests and your `controllers` also.

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @whiteWolfx99 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow Gitflow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -135,43 +121,36 @@ You can deploy this project using:
...
+- Original design idea by [Gregoire Vella on Behance](https://www.behance.net/gregoirevella).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (omarsalem7) [inline README.md:156]: - [ ] As stated by the previous reviewer, kindly make sure to use the main color like the design guidelines on all screens 
For example:

| Your design  | Design guidline |
| ------------- | ------------- |
| image  | image|

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @whiteWolfx99 😊, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

Almost near

 ## Highlights
✔️ Live demo works properly
✔️ Acknowledgments for author added

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Omar-Muhamad — APPROVED** [review]
# Approved ✔
### Hi @whiteWolfx99,

#### Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

### Congratulations! 🎉 

giphy (1)

## (Highlights) Good Points: :+1:
- Your work is well documented. ✔
- The correct workflow is used.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**